index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ahsan Ghazni Khan's Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <header>
+        <a href="">Ahsan Ghazni Khan</a>
+        <nav>
+            <button><img src="images/Icon-Menu.png" alt=""></button>
+        </nav>
+    </header>
+    <main>
+        <section class="hdln">
+            <h1>Hey There. I’m Ahsan Ghazni Khan</h1>
+            <h4>I’am a Software Developer</h4>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <ul class="sclmed">
+                <li><a href=""><img src="images/Icon -GitHub.png" alt=""></a></li>
+                <li><a href=""><img src="images/Icon - Linkedin.png" alt=""></a></li>
+                <li><a href=""><img src="images/Icon - Angelist.png" alt=""></a></li>
+                <li><a href=""><img src="images/Icon - Twitter.png" alt=""></a></li>
+                <li><a href=""><img src="images/Icon - Medium.png" alt=""></a></li>
+            </ul>
+        </section>
```

Comment:
- As the previous reviewer suggested, this section should have a background image as shown in the template. Please try to fix this. You may find this [article](https://www.w3schools.com/cssref/pr_background-image.asp) useful in using CSS to set background Images 👍 👍

Here's how your project should look like: 
![portfolio figma](https://user-images.githubusercontent.com/73607512/191563152-b040d54c-beb2-4b79-a127-7a32cf974582.PNG)

This is what your project currently looks like: 
![portfolio project](https://user-images.githubusercontent.com/73607512/191563415-40a20624-7b08-48c5-957d-8cbb43448d8f.PNG)

Reply:
but the image is being shown on my side
![Screenshot from 2022-09-21 21-53-56](https://user-images.githubusercontent.com/22774319/191565340-9074f4a6-5d7b-4f2b-8cf9-b06e36fc01a4.png)

Reply:
did you recheck it?

Reply:
what should I do next?

Reply:
I just confirmed the image is indeed there, but it is still not displaying on my browser. Please feel free to ask for another review. Sorry for any inconvenience 👋 👋